nextflow_schema.json

```diff
@@ -29,6 +29,11 @@
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "kraken2_db": {
+                    "type": "string",
```

Comment:
Could you add a `pattern` here to make sure the path has no whitespace (which might mess with how kraken2 is run). As in:

```json
"pattern": "^\\S+$",
```

Reply:
Added in [a2b375f](https://github.com/phac-nml/speciesabundance/pull/5/commits/a2b375f7eee1efa7fe4df93261daf1de19c1b096)